src/ingestion/calendar/scheduler.py

```
@@ -576,8 +582,8 @@ def sweep_value_side(
         When ``True`` (default) no HTTP and no DB writes. Connector
         dry-runs return their plan.
     start_year / end_year:
-        Year window passed through to BLS / BEA value-side ops (which
-        require it). Default: ``(current_year − 1, current_year)`` — a
+        Year window passed through to BLS / BEA / Census value-side ops.
+        Default: ``(current_year − 1, current_year)`` — a
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)